app/models/form526_submission.rb

```diff
@@ -60,6 +61,11 @@ def start
     jids.first
   end
 
+  def get_full_name
+    user = User.find(user_uuid)
```

Comment:
There's still a fairly high chance that get_full_name will return nil. `User.find` will only return a value if the user has an active session

Reply:
Ah, I see! Is it possible to submit the form without having an active session?